web/src/containers/Logout.js

```diff
@@ -8,15 +8,21 @@ import { logout as dispatchLogout } from '../actions/auth';
 
 const Logout = ({ authenticated, logout }) => {
   const history = useHistory();
-  useEffect(() => {
-    logout();
-  }, []);
+  useEffect(
+    () => {
+      logout();
+    },
+    // we want this to run on load so we don't need any thing
+    // in the dependency array
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    []
+  );
 
   useEffect(() => {
     if (!authenticated) {
       history.push('/login');
     }
-  }, [authenticated]);
+  }, [authenticated, history]);
```

Comment:
im curious, why run this effect on history change?

Reply:
I think it's because history is used in that use effect *and* it's a prop so it *could* change even though it can't really